frontend/src/components/Common/NewFunctionsLabel/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { NewFunctionsContent, NewLabel } from '@src/components/Common/NewFunctionsLabel/style';
+import { getVersion } from '@src/context/meta/metaSlice';
+import { useAppSelector } from '@src/hooks';
+import React, { ReactNode } from 'react';
+
+interface NewFunctionsLabelProps {
+  children: ReactNode;
+  style?: object;
+  createVersion: string;
+}
+
+export default function ({ children, style, createVersion }: NewFunctionsLabelProps) {
+  const version = useAppSelector(getVersion);
+
+  return (
+    <NewFunctionsContent>
+      {children}
+      {version <= createVersion && (
```

Comment:
`version <= createVersion` is correct? 
this comparison is based on the `string`， it is not workable

Reply:
done